setup.py

```diff
@@ -66,7 +66,7 @@
         "PyYAML",
     ],
     extras_require={
-        "train": ["tensorflow==2.2", "torch==1.3.0", "torchvision==0.4.1"],
+        "train": ["tensorflow==2.2.*", "torch==1.3.*", "torchvision==0.4.*"],
```

Comment:
While can we use a fixed version?

Reply:
I would prefer to use the newest revision of a version because it fixes issues with the version. If it is better to just lock it, that is fine too.

Reply:
I think locking the dependencies is necessary and important. Using a newer version brings us benefits but might also cause unexpected issues.

Reply:
BTW, speaking of dependency management, I have been thinking use [poetry](https://python-poetry.org/) as the management tool. Reasons:
1. Adopt PEP 518 (looks like a better dependency resolver? :smile: )
2.  Can still install poetry-managed project directly: `pip install -e .`

Reply:
OK, my thought was that `requirements.txt` is where we have the locked version. 

`ray[rllib]`, `tensorflow`, `torch` all use revision numbers for fixes to and since we have a test of `Setup.py` against `requirements.txt` we know when the version has changed and problems have occurred.